### ISSUE TITLE
[Question] Can this plugin automatically export all TypeScript types into Flow types?

### ISSUE THREAD

**Issue by zackarydev** [swansontec/rollup-plugin-flow-entry#21]
We have a repo in Flow that, and we would like to import a TypeScript package.

**Comment by swansontec** [comment]
Not really, since this is quite tricky! I actually work on another project where everything is in Flow, but we need TypeScript definitions for our external users.

To handle this, I actually [wrote a script](https://github.com/EdgeApp/edge-core-js/blob/4d78477a0472c368b72f97ca14e0646b715eebc6/scripts/make-types.js) for converting Flow to TypeScript. It's full of horrible regular expressions, but it does work for our case. If you look around on NPM, there are better solutions such as [FlowTS](https://github.com/zxbodya/flowts/tree/master).

Assuming you run a script like this as part of your build, you can then add a `types` property to your package.json file, pointing to the generated type definitions.
